nms/app/packages/magmalte/app/components/network/PoliciesConfig.js

```diff
@@ -151,11 +151,12 @@ function PoliciesConfig(props: WithAlert & {mirrorNetwork?: string}) {
         path={relativePath('/add')}
         component={() => (
           <PolicyRuleEditDialog
+            isV1={true}
```

Comment:
This change isn't necessary anymore right ?

Reply:
Right